packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -22,12 +22,21 @@ import { MeasurementPropertyWidget, MeasurementPropertyWidgetId } from "./Measur
 import { AbstractZoneLocation } from "@itwin/appui-abstract";
 import { UiFramework } from "@itwin/appui-react";
 
+interface Priority {
+  itemPriority?: number;
+  groupPriority?: number;
+}
+
 export class MeasureToolsUiItemsProvider implements UiItemsProvider {
   public readonly id = "MeasureToolsUiItemsProvider";
-  private _props?: { itemPriority: number };
+  private _props;
```

Comment:
you should type this.

also you should export the interface. 
also it just happens to be about priority right now, but it might expand later on to add other options. Something like MeasureToolsUiProviderOptions might be a better fit